lib/pod/command/trunk.rb

```diff
@@ -141,6 +141,17 @@ def time_zone
         out, = Executable.capture_command('/bin/date', %w(+%Z), :capture => :out)
         out.strip
       end
+
+      def netrc_path
```

Comment:
I tend to agree given that the code for handling GPG exists inside the upstream repo, just not for auto detecting it. I'll see if I can find any reasoning in commits/prs.